packages/helper-grammar-regex-collection/index.js

```diff
@@ -131,6 +131,21 @@ export const HTML_IMPORT = regex`
   href=${captureQuotedWord}>
 `;
 
+// handles the case the HTML script tag has a type with it
+const typeRegex = regex`
+  (\stype=['"][^'"\s]+['"])?
+`;
+
+export const HTML_SCRIPT_IMPORT = regex`
+  ^\s*
```

Comment:
I think we may want to delete this line, so that the regex matches `<script>` tags which aren't the first tag on their line.